src/Registration/Common/MutualInformation/MutualInformation.ipynb

```diff
@@ -0,0 +1,703 @@
+{
```

Comment:
<p>After the image is read in with <code>itk.imread</code>, we can get the dimension with:</p><p> </p><pre class="ql-syntax" spellcheck="false">dim = fixed_image.GetImageDimension()

</pre><p> </p><p>This way the script does not need to be hard coded for a dimension.</p><p> </p><p>The image type can be obtained with:</p><p> </p><pre class="ql-syntax" spellcheck="false">ImageType = type(fixed_image)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/InsightSoftwareConsortium/ITKExamples/pull/198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>